symfony/webpack-encore/0.9/webpack.config.js

```diff
@@ -0,0 +1,23 @@
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+    // directory where all compiled assets will be stored
+    .setOutputPath('web/build/')
+    // what's the public path to this directory (relative to your project's document root dir)
+    .setPublicPath('/build')
+    // empty the outputPath dir before each build
+    .cleanupOutputBeforeBuild()
+    // allow sass/scss files to be processed
+    .enableSassLoader()
```

Comment:
It uses `node-sass` - so it uses the C version of Sass. Still a problem @dunglas?